### DISCUSSION TITLE
When I submit a support ticket, I get an HTTP 500 error.

### DISCUSSION THREAD

**Discussion by Shad0wN1ght** [NXTS-Developers/MOFHY-Lite#75] [Q&A]
modules/Database/Config.php
I also entered the SQL information correctly, but I get an error message.

**Comment by greenreader9** [comment]
What error message? Are you setting it up using the "installation" files?

**Reply by Shad0wN1ght** [reply]
Yes,
https://i.ibb.co/0mgYb5V/rvdae-Jdims-UIEy-DK.png

**Reply by greenreader9** [reply]
That means the code crashed. Please try re-insalling

**Reply by Shad0wN1ght** [reply]
I reinstalled it, but that didn't fix it.

**Reply by greenreader9** [reply]
URL?

**Reply by Shad0wN1ght** [reply]
src/function/NewTicket.php
I get a code error somewhere in here.
I haven't even rewritten the code.

**Reply by mahtab2003** [reply]
You need to enable display errors in your cpanel

**Reply by Gouravyadavyt** [reply]
Where you host this site

**Reply by Shad0wN1ght** [reply]
The error log is as follows
Fatal error: Uncaught Error: Call to undefined function mysqli_real_escap_string() in /home/vol6_8/example.com/www/htdocs/src/function/NewTicket.php:7 Stack trace: #0 {main} thrown in /home/vol6_8/example.com/www/htdocs/src/function/NewTicket.php on line 7

**Reply by mahtab2003** [reply]
Error have been resolved

**Reply by Shad0wN1ght** [reply]
Thank you very much! It's working!
One more question, how do I set the characters in my email to UTF-8?

**Reply by mahtab2003** [reply]
Well I don't know about it either 😅

**Reply by Shad0wN1ght** [reply]
Thank you very much.
I solved the email problem by myself.
Where can I send a private message to mahtab2003? Preferably a chat app

**Reply by mahtab2003** [reply]
On whatsapp or email
